config/environment.js

```diff
@@ -16,12 +16,14 @@ if (environment === 'development') {
   require('dotenv').config()
 }
 
+const awsRegion = process.env.AWS_REGION || 'us-west-2'
```

Comment:
Should we set a default AWS_REGION in the helm chart (and remove all the js modifications in this PR)?